build/docker/zap-baseline.py

```diff
@@ -53,10 +53,12 @@
 import sys
 import time
 import traceback
-import urllib2
+from six.moves.urllib.parse import urlparse, urlencode
+from six.moves.urllib.request import urlopen
+from six.moves.urllib.error import HTTPError
 from datetime import datetime
 from random import randint
-from zapv2 import ZAPv2
+#from zapv2 import ZAPv2
```

Comment:
Was this changed on purpose? This will lead to errors, e.g.:
```
Unexpected error: <type 'exceptions.NameError'>
Traceback (most recent call last):
  File "zap-baseline.py", line 396, in main
    zap = ZAPv2(proxies={'http': 'http://' + zap_ip + ':' + str(port), 'https': 'http://' + zap_ip + ':' + str(port)})
NameError: global name 'ZAPv2' is not defined
```